db/warehouse/migrate/20230403144801_add_uploaded_by_to_files.rb

```diff
@@ -0,0 +1,5 @@
+class AddUploadedByToFiles < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :files, :uploaded_by, to_table: :User, null: true, index: true
```

Comment:
The new column should be `updated_by` since the `user_id` column already stores the uploader